backoffice/src/Services/MigrationService.cs

```diff
@@ -257,5 +260,33 @@ public async Task<WreckMaterial> MigrateWreckMaterialAsync(PowerappsWreckMateria
 
         return wreckMaterial;
     }
+
+
+    public async Task HandleTriageCsvAsync(List<TriageRowDto> records)
+    {
+        // loop through records
+        foreach ( var record in records )
+        {
+            // get report droitref
+            var droitReference = record.DroitReference;
```

Comment:
You can probably just use these straight from the record, rather than having them as new vars